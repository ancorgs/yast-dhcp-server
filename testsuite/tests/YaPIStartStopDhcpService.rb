# encoding: utf-8

module Yast
  class YaPIStartStopDhcpServiceClient < Client
    def main
      # testedfiles: DHCPD.pm

      Yast.include self, "testsuite.rb"

      Yast.import "YaPI::DHCPD"
      Yast.import "Mode"

      Mode.SetTest("testsuite")

      DUMP("==========================================================")
      TEST(lambda { YaPI::DHCPD.StartDhcpService({}) }, [], nil)
      DUMP("==========================================================")
      TEST(lambda { YaPI::DHCPD.StopDhcpService({}) }, [], nil)
      DUMP("==========================================================")
      TEST(lambda { YaPI::DHCPD.GetDhcpServiceStatus({}) }, [], nil)
      DUMP("==========================================================")

      nil
    end
  end
end

Yast::YaPIStartStopDhcpServiceClient.new.main
