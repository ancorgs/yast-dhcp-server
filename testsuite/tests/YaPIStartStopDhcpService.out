Dump	==========================================================
Execute	.target.bash "/etc/init.d/dhcpd restart" 0
Return	true
Dump	==========================================================
Execute	.target.bash "/etc/init.d/dhcpd stop" 0
Return	true
Dump	==========================================================
Execute	.target.bash "/etc/init.d/dhcpd status" 0
Return	true
Dump	==========================================================
